configure.ac

```diff
@@ -6962,6 +6962,67 @@ then
             [Define if you want to disable the GIL])
 fi
 
+# gh-109054: Check if -latomic is needed to get <pyatomic.h> atomic functions.
+# On aarch64, GCC may require programs to be linked explicitly to libatomic.
+# Use _Py_atomic_or_uint64() which may require libatomic __atomic_fetch_or_8()
+# on Linux aarch64 (or not depending on the compiler and the compiler flags).
+#
+# Avoid #include <Python.h> or #include <pyport.h>. The <Python.h> header
+# requires <pyconfig.h> header which is only written below by AC_OUTPUT below.
+# If the check is done after AC_OUTPUT, modifying LIBS has no effect anymore.
+# <pyport.h> cannot be included alone, it's designed to be included by
+# <Python.h>: it expects other includes and macros to be defined.
+LIBATOMIC=""
```

Comment:
I don't think is needed anymore :)

Reply:
What do you mean? Creating variables is not needed in configure.ac? I prefer to be explicit and declare it to an empty string.

Reply:
Oh, ``LIBATOMIC=${LIBATOMIC-"-latomic"}`` does nothing if LIBATOMIC is already defined (to an empty string). So I removed it.